plugin.video.digistorageplayer/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.digistorageplayer" name="Digi Storage Player" version="1.0.0" provider-name="RCSRDS">
+	<requires>
+		<import addon="xbmc.python" version="2.14.0"/>
+		<import addon="script.module.requests" version="2.9.1"/>
+	</requires>
+	<extension point="xbmc.python.pluginsource" library="addon.py">
+		<provides>executable</provides>
```

Comment:
do you want it listed in programs section? otherwise you could change it to "video".